src/pages/Products/Components/Category.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import './Category.scss';
+
+export default class Category extends Component {
+  state = {
+    categories: [],
+  };
+
+  componentDidMount() {
+    const url = '/data/category.json';
+    fetch(url)
+      .then(res => res.json())
+      .then(categories => this.setState({ categories }));
+  }
+
+  render() {
+    const { categories } = this.state;
+    const firstCategory = categories.map(category => {
+      return (
+        <option key={category.id} value={category.category}>
+          {category.category}
+        </option>
+      );
+    });
+    const secondCategory = categories.map(category => {
```

Comment:
```suggestion
    const mainCategories = categories.map(({ category, id }) => {
      return (
        <option key={id} value={category}>{category}</option>
      );
    });
    const subCategories = categories.map(category => {
```
- 변수명을 순서, 방향등으로 짓는 건 좋지 않아보입니다. 구체적인 내용을 반영할 수 있게 해주세요.
- 함수 파라미터 자리에서 곧바로 구조분해할당 가능합니다.

Reply:
대박! 지난번에 이어 구조분해할당 또 하나 배웠네요! 종택님 최고!!!